omod/src/main/webapp/resources/scripts/custom/deletePatient.js

```diff
@@ -0,0 +1,38 @@
+var delPatient = delPatient || {};
+
+delPatient.deletePatientCreationDialog = null;
+
+// meant to be overrideable
+delPatient.returnUrl = "/coreapps/patientdashboard/patientDashboard.page?patientId={{patientId}}&visitId={{visitId}}";
+
+/**
+ * Functions used to Delete a Patient (uses QuickVisitFragmentController)
+ */
+
+delPatient.createDeletePatientCreationDialog = function() {
+    delPatient.deletePatientCreationDialog = emr.setupConfirmationDialog({
+        selector: '#delete-patient-creation-dialog',
+        actions: {
+            confirm: function() {
+                emr.getFragmentActionWithCallback('coreapps', 'visit/quickVisit', 'create',
+                    { patientId: delPatient.patientId, locationId: sessionLocationModel.id() },
+                    function(v) {
+                        jq('#quick-visit-creation-dialog' + ' .icon-spin').css('display', 'inline-block').parent().addClass('disabled');
+                        delPatient.goToReturnUrl(delPatient.patientId, v);
+                    });
+            },
+            cancel: function() {
+                delPatient.deletePatientCreationDialog.close();
+            }
+        }
+    });
+
+    delPatient.deletePatientCreationDialog.close();
+}
```

Comment:
I noticed this in the code. Why do some functions not have ending semi-colons while some do? I copied the exact template from the 'QuickVisits' thing so idk